src/tree/AttachedAccountsTreeItem.ts

```diff
@@ -251,15 +252,22 @@ export class AttachedAccountsTreeItem extends AzureParentTreeItem {
     }
 
     private async canConnectToLocalMongoDB(): Promise<boolean> {
-        try {
-            const db = await connectToMongoClient(localMongoConnectionString, appendExtensionUserAgent());
-            // grandfathered in
-            // tslint:disable-next-line: no-floating-promises
-            db.close();
-            return true;
-        } catch (error) {
+        async function timeout(): Promise<boolean> {
+            await delay(1000);
             return false;
         }
+        async function connect(): Promise<boolean> {
+            try {
+                const db: MongoClient | void = await connectToMongoClient(localMongoConnectionString, appendExtensionUserAgent());
```

Comment:
Haha I hope you tested the code because I didn't - also I messed around with a few options and must've copied something unintentionally because this `| void` thing doesn't need to be there

Reply:
Lol I didn't notice the void. No worries, I tested it, it's working.